src/sass/textbox/textbox.scss

```diff
@@ -193,10 +198,18 @@ textarea.textbox__control {
     }
 }
 
-.textbox--large > input.textbox__control {
+.textbox--large {
     height: 48px;
 }
```

Comment:
This shouldn't be necessary anymore. You can remove it.

Reply:
I think we still need it for large, right? I changed it to use props.

Reply:
We shouldn't, because you are setting the `.textbox__control` size to 46px

Reply:
I think that height is just inferred though, which makes it slightly more brittle, but I'm fine either way.

Reply:
I'll remove it to match default. We can deal with it if there's additional issues.